solutions/standard/variables.tf

```diff
@@ -236,6 +246,12 @@ variable "ibmcloud_kms_api_key" {
 # Event Notifications
 ########################################################################################################################
 
+variable "enable_event_notification" {
```

Comment:
@in-1911 I believe we add all variables in our catalog pipeline, and only need to put it in catalog json if we wish it to be hidden, or to enable special field like a dropdown (like for regions)

Reply:
I am verifying this before merge though

Reply:
@in-1911 I've confirmed that ibm_catalog.json variables are indeed optional if there are no differences from terraform, however...

It can affect the order/display of the variables in the UI in projects. If a variable is not in the catalog json it will most likely get added to the bottom of the list of optionals. So if we want this new "enable" field to sit right above or below the "existing_event_notification_crn" in the UI, we would need to add it.

Should we do that now?

Reply:
@toddgiguere - at least from my perspective the order and placement is not that important (as long as it shows up in the UI and/or can be set through an environment parameter) , so I'd rather get the new release published sooner.